nix/tests/lanzaboote.nix

```diff
@@ -338,16 +341,28 @@ in
   };
 
   # We test if we can install Lanzaboote without Bootspec support.
-  synthesis = mkSecureBootTest {
-    name = "lanzaboote-synthesis";
-    machine = { lib, ... }: {
-      boot.bootspec.enable = lib.mkForce false;
-    };
-    testScript = ''
-      machine.start()
-      assert "Secure Boot: enabled (user)" in machine.succeed("bootctl status")
-    '';
-  };
+  synthesis =
+    if pkgs.hostPlatform.isAarch64 then
```

Comment:
Not sure what the issue is here but since it's already broken, it could be maybe fixed by someone else who knows the bootspec better?

Reply:
I never really looked into synthesis. I'll let @RaitoBezarius take the call on this.

Reply:
I know what's going on, but it's not needed to block on it. We fixed the synthesis on aarch64 but I'm not sure we cut another release in nixpkgs for it.